src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -167,10 +200,32 @@ def main():
     break_network_at_waterbodies = args.break_network_at_waterbodies
     csv_output_folder = args.csv_output_folder
     assume_short_ts = args.assume_short_ts
-
+    custom_input_file = args.custom_input_file
     test_folder = pathlib.Path(root, "test")
     geo_input_folder = test_folder.joinpath("input", "geo")
 
+    if custom_input_file:
+        (
+            supernetwork_parameters,
+            waterbody_parameters,
+            forcing_parameters,
+            restart_parameters,
+            output_parameters,
+            run_parameters,
+        ) = nhd_io.read_custom_input(custom_input_file)
+
+        qlat_const = forcing_parameters.get("qlat_const", None)
+        qlat_input_file = forcing_parameters.get("qlat_input_file", None)
+        qlat_input_folder = forcing_parameters.get("qlat_input_folder", None)
+        qlat_file_pattern_filter = forcing_parameters.get(
+            "qlat_file_pattern_filter", None
+        )
+    else:
```

Comment:
I think we can leave this out of the if-block for this PR and just use the command-line inputs.

Reply:
Won't the end result be using a custom input file anyway? I figured right now it loads in and we can take what we want from it and disregard the rest. Wouldn't command-line be going backward?